plugins/catalog-backend-module-openapi-spec/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-openapi-spec",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
Intentional? Fine by me anyway if we consider this to be very experimental, especially given the naming discussion

Reply:
Unintentional, as this was the default when I created, will update